### PR TITLE
make overview card more responsive. removed use of Media

### LINKED ISSUE #946: Make dashboard more responsive
It looks like the VM name and status get lost once the screen reaches a certain size however and that seems like important information to still have available. My one question is would it make sense to put the action buttons into a kebab menu if the screen size reaches a certain, narrow size?
https://github.com/oVirt/ovirt-web-ui/pull/934#pullrequestreview-198140711

@gregsheremeta @michalskrivanek @sjd78 @lwrigh

### PR DESCRIPTION
**Pull request by thaorell**
Fix #946 
Before:
screenshot-localhost_3000-2019 07 24-15_51_27

After: 
screenshot-localhost_3000-2019 07 24-15_50_01

### CODE REVIEW

File: src/components/VmDetails/cards/OverviewCard/style.css [hunk 1]
```diff
@@ -46,3 +46,14 @@
   line-height: 20px;
   padding-left: 10px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .media-left {
```
Comment (sjd78) [inline src/components/VmDetails/cards/OverviewCard/style.css]:
```suggestion
  :global(.media-left) {
```
Then you can drop the `className` attribute on the `Media.Left` tag

File: src/components/VmDetails/cards/OverviewCard/style.css [hunk 2]
```diff
@@ -46,3 +46,14 @@
...
+    display: block;
+  }
+  
+  .media-body {
```
Comment (sjd78) [inline src/components/VmDetails/cards/OverviewCard/style.css]:
```suggestion
  :global(.media-body) {
```
Then you can drop the `className` attribute on the `Media.Body` tag.

**Comment by sjd78** [comment]
ci test please

**Comment by isaranova** [comment]
ci build please

**Comment by isaranova** [comment]
Text from overview is not moving under the picture if the windows size is small.

Screenshot_2019-08-26 VM Portal(1)
firefox-68.0-4.fc30.x86_64

**Comment by thaorell** [comment]
@isaranova what was the tested resolution?

**Comment by isaranova** [comment]
@thaorell around 300x900px

**Comment by isaranova** [comment]
ci build please

**Comment by isaranova** [comment]
LGTM

**Pull request merged**